tests/setup.js

```diff
@@ -62,6 +62,16 @@ if (global.Intl) {
   global.Intl = require('intl');
 }
 
+if (!global.matchMedia) {
```

Comment:
Required by? `react-slick`?

Reply:
Yeah. But we aren't using it anymore so yay!